types/colornames/colornames-tests.ts

```diff
@@ -1,19 +1,21 @@
 import colorNames = require("colornames");
 
-// $ExpectType Color
+// $ExpectType string | undefined
 const color = colorNames("red");
-
-// $ExpectType string
-color.value;
+// $ExpectType string | undefined
+color;
+// $ExpectType string | undefined
+const undefinedColor = colorNames("donkey");
+// $ExpectType string | undefined
+undefinedColor;
```

Comment:
You may want to replace this portion of code by
```ts
// $ExpectType string | undefined
colorNames("red");
// $ExpectType string | undefined
colorNames("donkey");
```

As checking the variables a second time is not necessary.

Reply:
@manuth thanks for the feedback!